packages/coinjoin/package.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "@trezor/coinjoin",
+    "version": "1.0.0",
+    "private": true,
+    "author": "Trezor <info@trezor.io>",
+    "homepage": "https://github.com/trezor/trezor-suite/packages/coinjoin",
+    "description": "Coinjoin client",
+    "repository": {
+        "type": "git",
+        "url": "git://github.com/trezor/trezor-suite.git"
+    },
+    "bugs": {
+        "url": "https://github.com/trezor/trezor-suite/issues"
+    },
+    "license": "See LICENSE.md in repo root",
+    "sideEffects": false,
+    "main": "lib/index.js",
+    "files": [
+        "lib/",
+        "!**/*.map"
+    ],
+    "types": "lib/index.d.ts",
+    "scripts": {
+        "lint": "eslint '**/*.{ts,js}'",
+        "test:unit": "jest",
+        "type-check": "tsc --build",
+        "build:lib": "rimraf lib && tsc --build ./tsconfig.lib.json"
```

Comment:
not sure if this is final version, but with current usage build seems unecessary

Reply:
this lib will be used on electron main side so needs to be build
implementation is on the way :)